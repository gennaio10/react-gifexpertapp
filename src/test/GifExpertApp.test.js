
import React from "react";
import '@testing-library/jest-dom';
import GifExpertApp from "../GifExpertApp";
import { shallow } from "enzyme";

describe("Pruebas de <GifExpertApp />)", () => { 

  test(' debe mostrar el componenete correctamente', () => {
    const wrapper = shallow(<GifExpertApp />);
    expect(wrapper).toMatchSnapshot();
  });

});
