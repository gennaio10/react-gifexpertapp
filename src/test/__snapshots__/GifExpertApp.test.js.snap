// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de <GifExpertApp />)  debe mostrar el componenete correctamente 1`] = `
<Fragment>
  <h2>
    Gif Expert App
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="Goku"
      key="Goku"
    />
  </ol>
</Fragment>
`;
